### ISSUE TITLE
The home page needs to not look terrible.

### ISSUE THREAD

**Issue by MatthewJamesBoyle** [Golang-Insiders/site#4]
We should add some styling to it and keep it consistent across pages. Not worried about it being perfect, we can improve this over time.

**Comment by kingstarr** [comment]
I have a template I bought a few years ago, I could adopt it for the site, and write a Golang backend to serve it some functionalities.

If this is approved.

**Comment by MatthewJamesBoyle** [comment]
Could you provide more details
On the the template please? Would be good to take a look and ensure the licenses are ok for our use

**Comment by kingstarr** [comment]
Alright Matt,

Below is a link to preview:

http://preview.themeforest.net/item/invetex-business-consulting-investments-site-template/full_screen_preview/18318575?_ga=2.2510640.1866831668.1701175700-386922872.1701175700

I can provide you with login details to access the product download, if you require that.

I look forward to your response.

Kingsley Ifedayo

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by MatthewJamesBoyle** [comment]
It looks kind of formal for a community site I think? 
I like the playfulness of the Gophercon Signapore site: https://2023.gophercon.sg

maybe this template can be adapted. What do you think ?

**Comment by kingstarr** [comment]
It is beautiful, and the simplicity is alluring.

I would adopt it over mine. I believe we should adapt it. How do I proceed with my contribution?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by namkatcedrickjumtock** [comment]
@MatthewJamesBoyle are there any other issues that haven't been picked up?

**Comment by MatthewJamesBoyle** [comment]
@kingstarr  assigned this one to you! Looking forward to seeing a PR

**Comment by MatthewJamesBoyle** [comment]
@namkatcedrickjumtock https://github.com/Golang-Insiders/site/issues/1 hasn't been picked up yet. Also feel free to raise suggestions for other things the site could have and we can discuss them. You can then work on them :)

**Comment by namkatcedrickjumtock** [comment]
got it :+1:

**Comment by kingstarr** [comment]
Thank you Matt,

I will get to work. If you are aware of any resource I could use please let me get the link; such as the link to the template.

By the way, what is PR?

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by MatthewJamesBoyle** [comment]
@kingstarr there isn't a template, we'd need to design something ourselves. 

PR is an abbreviation for Pull Request. 

If this task is a little too unclear don't worry, someone else can work on this one and we can find something a little more concrete for your first contribution.

**Comment by kingstarr** [comment]
Yes, I do more of backend tasks than frontend.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by MatthewJamesBoyle** [comment]
@namkatcedrickjumtock you can take this one if you want :)

**Comment by MatthewJamesBoyle** [comment]
just a quick bump @namkatcedrickjumtock

**Comment by namkatcedrickjumtock** [comment]
alright sure I can work on it, got some experience with frontend, just to confirm some couple of things before i start working on it @MatthewJamesBoyle .

- My understanding is we ll need to design the homepage ourselves something similar to the [Gophercon Signapore site](https://2023.gophercon.sg/) since we don't have design yet correct?
- it is at my discretion to use technologies like `tailwind css` for styling of the site or do we need another repo setup for the frontend app using NextJs framework , what'd you think?

**Comment by MatthewJamesBoyle** [comment]
@namkatcedrickjumtock  yeah that all sounds great. Its mostly just to get the ball rolling - I don't mind about the design being perfect or anything :)

**Comment by namkatcedrickjumtock** [comment]
alright, copy that, I ll setup a NextJs app for the frontend & open a PR if it's looks like something we could work with then we ll fork it into Insider Organisation. :+

**Comment by MatthewJamesBoyle** [comment]
We shouldn’t need a next js app, you should be able to do it in this repo in Go.

**Comment by namkatcedrickjumtock** [comment]
alright, got it 👍

**Comment by namkatcedrickjumtock** [comment]
@MatthewJamesBoyle heads up  it seems i don't have permission to open PRs for this repo.
